File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    printf("Height: ");
    int height = get_int();
    
    if(height < 1 || height > 23) {
        printf("Please enter a positive number no greater than 23\nRetry: ");
        height = get_int();
    }
    for(int i = 1; i <= height; i++){
        for(int j = 0; j < height - i; j++){
            printf(" ");
        }
        for(int j = 0; j < i; j++){
            printf("#");
        }
        printf("  ");
        for(int j = 0; j < i; j++){
            printf("#");
        }
        for(int j = 0; j < height - i; j++){
            printf(" ");
        }
        printf("\n");
    }
    
}

File: pset6/mario/mario.py
print("Height: ", end="")
height = int(input()) + 1

while height < 1 or height > 23:
    print("Please enter a positive number no greater than 23\nRetry: ", end="")
    height = int(input())

for i in range(height):
    for j in range(height-i):
        print(" ", end="")
    for j in range(i):
        print("#", end="")
    print("  ", end="")
    for j in range(i):
        print("#", end="")
    for j in range(height-i):
        print(" ", end="")
    print("");